app/models/invenio_rdm_record_converter.rb

```diff
@@ -170,4 +170,16 @@ def subjects_for_scheme(terms, scheme)
       terms.map{ |term| {subject: term} }
     end
   end
+
+  def additional_titles(titles)
+    additional_titles_size = titles.size-1
+    return nil if additional_titles_size < 0
```

Comment:
It's annoying, but I did not want to use `shift` on the arrays for `description` and `title` here since that would change the actual structure of the array. By using this sort of messing select nothing is changed with the underlying array.